src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -232,6 +244,16 @@
                 namespace,
                 (state: ADRState) => state.adrError
             ),
+            releaseName(){
+                return this.selectedRelease?.name || null;
+            },
+            releaseURL(){
+                if (this.selectedDataset?.url){
+                    return this.selectedRelease?.activity_id ? new URL(this.selectedDataset.url).origin + '/dataset/' + this.selectedDataset.title.replace(/\s+/g,"-").toLowerCase() + '?activity_id=' + this.selectedRelease.activity_id : "";
```

Comment:
```suggestion
                if (this.selectedDataset && this.selectedRelease) {
                    return new URL(this.selectedDataset.url).origin + '/dataset/' + this.selectedDataset.id + '?activity_id=' + this.selectedRelease.activity_id;
```

Reply:
You've put me in a bit of bind for writing code coverage here as there is no (even theoretical) situation where the url is rendered with empty strings and the else side of this statement is reached.

Reply:
Surely it can be removed then?